app/controllers/merchants_controller.rb

```diff
@@ -4,6 +4,10 @@ def index
   end
 
   def show
-    @merchant = Merchant.find(params[:id])
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def merchant_params
```

Comment:
I dont think merchant params is used elsewhere, it should probably fall under private methods, so add `private` on line 9.5

Reply:
Good catch